src/Microsoft.AspNet.Mvc.Core/ActionResults/ObjectResult.cs

```diff
@@ -38,6 +38,18 @@ public ObjectResult(object value)
         /// </summary>
         public int? StatusCode { get; set; }
 
+        /// <summary>
+        /// Gets or sets the flag to exclude formatters that match only on the object type.
```

Comment:
Probably something like "Gets or sets a flag to enable content-negotiation to select a formatter based on object type." ?